demo/package.json

```diff
@@ -8,5 +8,8 @@
       "packagesFolder": ".",
       "deepLevel": 3
     }
+  },
+  "dependencies": {
+    "undefined": "^0.1.0"
```

Comment:
Could you remove that dependency? As far as I know, it's a known issue at sui-mono.

Reply:
Yes, sorry